core/storage/voiceover/gae_models.py

```diff
@@ -162,7 +162,7 @@ def create_new(
         entity_version: int,
         language_accent_code: str,
         voiceovers_mapping: Dict[str, Dict[
-            feconf.VoiceoverType.value, Optional[state_domain.VoiceoverDict]]],
+            str, Optional[state_domain.VoiceoverDict]]],
```

Comment:
Why are we changing this? Probably better to tighten the type to specific strings, right?